### PR TITLE
Add 15 docs for series func

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

**Comment by AlexanderKalistratov** [comment]
@1e-to CI failed & pep issues

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -1653,9 +1675,33 @@ def hpat_pandas_series_copy_impl(self, deep=True):
 @sdc_overload_method(SeriesType, 'corr')
 def hpat_pandas_series_corr(self, other, method='pearson', min_periods=None):
     """
-    Pandas Series method :meth:`pandas.Series.corr` implementation.
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
 
-    Note: Unsupported mixed numeric and string data
+    Pandas API: pandas.Series.corr
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Please add limitations section and specify what only `method='pearson'` is supported
Also please add check for the method

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 2]
```diff
@@ -2247,10 +2294,36 @@ def hpat_pandas_series_add_number_impl(self, other, level=None, fill_value=None,
 @sdc_overload_method(SeriesType, 'sub')
 def hpat_pandas_series_sub(self, other, level=None, fill_value=None, axis=0):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.sub
+
+    Examples
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Limitation section stated that `fill_value` is not supported

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 3]
```diff
@@ -2404,34 +2491,68 @@ def hpat_pandas_series_sum_impl(
 @sdc_overload_method(SeriesType, 'take')
 def hpat_pandas_series_take(self, indices, axis=0, is_copy=False):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.take
+
+    Examples
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: limitation section about `is_copy`

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 4]
```diff
@@ -2832,10 +2938,36 @@ def hpat_pandas_series_truediv_number_impl(self, other, level=None, fill_value=N
 @sdc_overload_method(SeriesType, 'floordiv')
 def hpat_pandas_series_floordiv(self, other, level=None, fill_value=None, axis=0):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.floordiv
+
+    Examples
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Limitation section about `fill_value`

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 5]
```diff
@@ -2766,7 +2877,7 @@ def hpat_pandas_series_truediv(self, other, level=None, fill_value=None, axis=0)
     Pandas Series :meth:`pandas.Series.truediv` implementation.
     .. only:: developer
 
-       Test: python -m sdc.runtests sdc.tests.test_series.TestSeries.test_series_op5
+       Test: python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_op5*
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Limitation section about `fill_value`

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 6]
```diff
@@ -2669,9 +2786,9 @@ def hpat_pandas_series_div(self, other, level=None, fill_value=None, axis=0):
     Intel Scalable Dataframe Compiler Developer Guide
     *************************************************
     Pandas Series method :meth:`pandas.Series.div` implementation.
-    .. only:: developer
 
-       Test: python -m sdc.runtests sdc.tests.test_series.TestSeries.test_series_op5
+    .. only:: developer
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Limitation section about `fill_value`

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 7]
```diff
@@ -3439,71 +3627,78 @@ def hpat_pandas_series_mean_impl(self, axis=None, skipna=None, level=None, numer
 @sdc_overload_method(SeriesType, 'mod')
 def hpat_pandas_series_mod(self, other, level=None, fill_value=None, axis=0):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.mod
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Limitation section about `fill_value`

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 8]
```diff
@@ -1676,9 +1722,10 @@ def hpat_pandas_series_corr(self, other, method='pearson', min_periods=None):
     Returns
     -------
     :obj:`float`
-         returns :obj:`float` object
+        returns :obj:`float` object
     """
 
+    _func_name = 'Method corr().'
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Why did you add the line? It seems unnecessary.

File: sdc/datatypes/hpat_pandas_series_functions.py (hunk 2 above)
Comment (1e-to) [reply sdc/datatypes/hpat_pandas_series_functions.py]: Now the not implement parameters are described in the notes section. Need to remove them to limitations?

File: sdc/datatypes/hpat_pandas_series_functions.py (hunk 8 above)
Comment (1e-to) [reply sdc/datatypes/hpat_pandas_series_functions.py]: This is necessary so that all functions are in the same style.

Comment (densmirn) [reply sdc/datatypes/hpat_pandas_series_functions.py]: Now I see it's used and necessary.
